bin/pc2installservice

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+#
+# Purpose:
+#   This script installs a Linux service that runs the script to create
```

Comment:
The English grammar here is confusing:  "this script installs a Linux service to run the script..."   On first reading it sounds like "This script installs a Linux service to run this script".  It would be clearer if it said something like "This script installs a Linux service that runs the `pc2installsandbox` script to create the the cgroup necessary..."  (I know this is minor, and I wouldn't withhold approval of the PR for this.  I just don't like having to stumble and read explanatory comments multiple times trying to parse their meaning....)

Reply:
I can combine that and the next sentence after it, which tells you which script it runs.

Reply:
Again, it's a minor issue, but I didn't see any change related to the above.  Instead of "this script installs a Linux service to run the script..."  I'd like it to say (something like)  "This script installs a Linux service which, on each reboot, runs the script `pc2installsandbox` which in turn installs the CGroup necessary to support..."